src/typescript/4-types.md

```diff
@@ -18,54 +18,88 @@ The exercises will teach you how to:
 - Spot and correct basic type mistakes
 - Create a typed variable
 
+## Basic Types
 
+Types explain what you can and can’t do with a value, take for example numbers and strings. In JavaScript, on strings, you have access to `.length`, on a number you do not. Types also determine what will happen when you apply operators to values. For example, the `+` operator, what does the following do?
 
-## Basic Types
+```javascript
+console.log(a + b);
+```
+
+Some might say it adds `a` and `b` together, others that it concatenates `a` and `b`. Both are potentially right. To come to the correct answer, we need to know what `a` and `b` are. If `a` and `b` are numbers, the `+` will perform addition, if they are strings - concatenation. The difference between the two is their underlying types.
 
-Types are what determine how much memory is allocated to save a value. Types also determine what operations or methods can be performed on a value. Types like strings can't have mathematical operations performed on them, but methods like ``.length`` can be used.
+> **Note:** If you are familiar with JavaScript’s primitive types you may recognize some of the names coming up. The types `string` `boolean` and `number` should feel familiar. These types are the same in TypeScript as they are in JavaScript and are at times referred to as primitive types.
 
 ### Boolean
 
+`boolean` is a type that can only be `true` or `false`.
+
 ```typescript
 let isCarnivore: boolean = true;
+let isHerbivore: boolean = false;
 ```
 
 ### Number
 
+`number` is used for numbers. If you are familiar with other typed languages, like `c++` you may be familiar with specific number types like `unsigned int` and `float`. In JavaScript and TypeScript, there is only `number`.
+
 ```typescript
 let teeth: number = 100;
-let hex: number = 0xF00D;
+let hex: number = 0xf00d;
 ```
 
 ### String
 
+`string` is used for string values.
```

Comment:
using the word in the definition?

Reply:
OOoooo thanks for catching that. I updated it a bit to be something along the lines of a collection of characters